### ISSUE TITLE
"Duplicate" patch being applied for ReStock Plus Cherenkov when 'Other Mod Support' patches are installed

### ISSUE THREAD

**Issue by Poodmund** [post-kerbin-mining-corporation/KerbalAtomics#90]
As https://github.com/ChrisAdderley/KerbalAtomics/blob/master/GameData/KerbalAtomics/Patches/NTR/hydrogenNTRsRestockPlus.cfg is bundled in with the main core distribution, when a user also installs the 'Other Mod Support' patches and has ReStock Plus installed, https://github.com/ChrisAdderley/KerbalAtomics/blob/master/Extras/KerbalAtomicsLH2NTRModSupport/hydrogenNTRsRestockPlus.cfg also gets applied to the Cherenkov engine. This causes 2 engine mode switch modules to get applied to the engine. See below:

![Untitled-1](https://user-images.githubusercontent.com/10106634/98462492-acac5a00-21ac-11eb-9b08-6f540968dd52.png)

Comparison of the two config files below:

![Untitled-2](https://user-images.githubusercontent.com/10106634/98462503-b766ef00-21ac-11eb-808a-c48f2f5f1ede.png)

Is it best to consolidate the correct config patches to https://github.com/ChrisAdderley/KerbalAtomics/blob/master/Extras/KerbalAtomicsLH2NTRModSupport/hydrogenNTRsRestockPlus.cfg and remove https://github.com/ChrisAdderley/KerbalAtomics/blob/master/GameData/KerbalAtomics/Patches/NTR/hydrogenNTRsRestockPlus.cfg that is bundled with the core mod?

**Comment by hemeac** [comment]
@ChrisAdderley, do you have any strong feelings about the solution to this?  Happy to submit a PR, I've had a few people use this with KTT which causes an issue with B9 as this leads to the Cherenkov having two ModuleEnginesFX with engineID = LF which I cannot easily distinguish.  Rather have the solution come by patching this rather than doing a workaround in KTT.

**Comment by ChrisAdderley** [comment]
Uh I guess i have to fix this sometime, sorry, didn't really realize this was urgent

**Comment by hemeac** [comment]
Not super urgent, was just reminded of this as I was going through my to do list of things for bug fixing.

**Comment by Wyzard256** [comment]
I just noticed this too.  Looking at the change history, I see that the file from Patches/NTR was moved out to Extras in commit adf2ff7fd4907faaa83c876ea346581e244995a9, then a new copy of the file was created in Patches/NTR in commit 6529987812736868e0826bb8cbf0c8ce370ecb9d.

The older copy of the file (the one that's now in Extras) has a comment at the top that says it's for the Nerv (it clearly began as a copy of the actual Nerv patch), and the newer copy (in Patches/NTR) correctly says it's for the Cherenkov, so it looks like that one is the newer _version_ of the patch, development-wise (in addition to being the more-recent commit).  The one in Extras should probably just be removed.
